string lesson_name
---
string message
